### ISSUE TITLE
New user, created through mobile Web, can’t log into app

### ISSUE THREAD

**Issue by Basil-1** [bluesky-social/social-app#6985]
### Steps to Reproduce

As a new user, using the same iphone I used to create my account, I can log into bluesky using any mobile browser (tested with safari and brave, including in privacy mode). I can not, however, log into the app on the same phone. Nor can I use the credential to log into the site from a windows computer (tested with firefox and edge).

### Attachments

https://github.com/user-attachments/assets/2e4b188a-b2a3-467e-a9b6-1307950aa304

### What platform(s) does this occur on?

iOS, windows 10

### Device Info

iOS 18.1, iPhone 16. Windows 10

### What version of the app are you using?

1.95

**Comment by gaearon** [comment]
The problem is you need to enter your user handle into that field, not the email.

**Comment by Basil-1** [comment]
The problem persists- this time I used a browser on windows to test:

![image](https://github.com/user-attachments/assets/334193c1-5177-4892-b64d-c6f693839c45)

**Comment by gaearon** [comment]
Have you tried going through the “forgot password” flow?

**Comment by Basil-1** [comment]
Yes once. I also set up 2FA and that works when I use the browser on the phone
